src/features/expressions/shared-tests/functions/externalApi/should-pick-data-with-nested-path.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Should pick data with nested path",
+  "expression": ["externalApi", "testId", "results[0].id"],
+  "expects": "1",
+  "externalApis": { "testId": { "data": { "results": [{ "id": "1" }] } } }
```

Comment:
Any reason this is packed inside `data` and `results`? :thinking: Can there be more information outside `data`? If it's a direct mapping to the results from `useQuery()`, we can't really tie this all to a frontend-only tech, as these expressions should run on the backend as well.

Also, from reading this and other tests, it kindof looks like all these looks have to start with `results...`? I would think less generically-sounding test data would resolve my misunderstanding, i.e. if the data were something like `{ "Cars": [{ "Brand": "BMW" }, { "Brand": "Lada" }] }`